### PR TITLE
Additions to snippets for lore and flavor

### PR DESCRIPTION
**Pull request by rtizzy**
#### Summary
```SUMMARY: Content "Additional snippets for lore"```
#### Purpose of change

https://github.com/CleverRaven/Cataclysm-DDA/issues/24278

Partial remediation of above mentioned issue

#### Describe the solution
Simply adds additional notes to the game to improve flavor and lore. 

#### Additional context

This was a project originally started in 2013. 

The game currently lacks any significant in game lore. 

The series of PRs related to this issue are intended to improve and increase the amount of lore available to the player character in the game. 

These PRs would be far more effective if the following issue is addressed. 

https://github.com/CleverRaven/Cataclysm-DDA/issues/28853

In short: Allowing a more immersive way of viewing in game items such as notes, journals, and newspapers.

### CODE REVIEW

File: data/json/snippets/snippets.json [hunk 1]
```diff
@@ -1803,6 +1803,54 @@
...
+    "category": "note",
+    "text": ["Dear John,\nI want you to know I intended to wait until you returned from Korea, but as time has gone it’s become to be more than I can take. From what they’re saying on the news it seems you’ll be gone longer than was first thought. I sometimes stay up at night wondering if you will ever make it home at all and it’s just so much to think about.\nI'm sorry, but this just isn't working. I know you never chose to be drafted and that you would be by my side if you could. I want you to know that it’s not your fault but I’m at a time in my life where I need someone here to support me. I know you must be angry, but I know you’ll understand. I hope you get home safe John.\nSincerely,\nJen\""
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": ["\"WOLF ATTACKS ON THE RISE!\nIncreasing reports across the New England area of wolves attacking local wildlife, and even people. Forest rangers are baffled with the oddly aggressive behaviour, and while they have not yet determined the cause, they advise anyone sighting a wolf to avoid contact. If threatened - fight back. Running seems to provoke them to attack.\""
```
Comment (I-am-Erk) [inline data/json/snippets/snippets.json]: This looks like it's meant to be a newspaper headline, not a survivor note.

File: data/json/snippets/snippets.json [hunk 2]
```diff
@@ -1803,6 +1803,54 @@
...
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": ["\"NEW LINE OF CBMS RELEASED!\nGE are proud to announce the latest line of CBMS, the Metabolic Exchange System! Now your batteries can be powered from the food you eat, no more do you have to rely on expensive energy prices! Available now - while stocks last!\n(GE is not responsible for starvation that may or may not be caused by this system. If symptoms of hunger or lower power persist after eating, please consult your local doctor)\""
```
Comment (I-am-Erk) [inline data/json/snippets/snippets.json]: This looks like it's meant to be a flier, not a note.
I'd be slightly cautious about adding too many CBM based advertisements as we're still figuring out the timeline on what CBMs were available to the public... this one should be safe, but it's an area under review.

File: data/json/snippets/snippets.json [hunk 3]
```diff
@@ -1803,6 +1803,54 @@
...
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": ["\"VAC AND RIVTECH WEAPON MANUFACTURERS AT IT AGAIN!\nThe two biggest names in military and civilian firearms are back at their competitive antics. After a court meeting between the two companies that left VAC suing for “intellectual theft”, Rivtech was reportedly exposed in an attempt to sabotage a Virgil Armament Company manufacturing plant. It is unsure if Rivtech will be facing legal consequences, and both companies refuse to comment.\""
```
Comment (I-am-Erk) [inline data/json/snippets/snippets.json]: This looks like it's meant to be a newspaper clipping not a note.

I think that VAC and RivTech are probably "two of the big names", but probably not the biggest. Up to you if you want to change it, might be a sensational reporter.

File: data/json/snippets/snippets.json [hunk 4]
```diff
@@ -1803,6 +1803,54 @@
...
+    ]
+  },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": ["\"STRANGE EVENTS IN SMALL NEW ENGLAND TOWN\nAccording to residents of a small New England town, a local science laboratory has been keeping them up at nights. Many people spoken to complain about loud noises and flashes of light coming from a government sanction lab outside of town. Other eyewitnesses suggest seeing “crazed, naked men” in the woods around the lab. Government officials withheld any comment on these occurrences, suggesting that the noises and flashes of light are coming from military training routines. They refused to talk anymore when we mentioned the sightings of the crazed, naked men.\""
```
Comment (I-am-Erk) [inline data/json/snippets/snippets.json]: Check punctuation, you've got a `“ ”` instead of a pair of ` \"`.
Looks like another news article, not note.

File: data/json/snippets/snippets.json [hunk 5]
```diff
@@ -1803,6 +1803,54 @@
...
     ]
   },
+  {
+    "type": "snippet",
+    "category": "note",
+    "text": ["\"Whoever finds this, listen to these words. This is what has kept me alive this long. If I’m not back in a day though, this isn’t enough. Burn some of those walking corpses in my memory. \n1. Zombies hate noise, they'll try to kill you if you make it, but enough noise will kill THEM\n2. The walking plant things are allergic to ammonia\n3. Zombies can’t smell you through smoke!\n4. Fire kills EVERYTHING\n5. Zombies can't smell OR see you if you're high on meth! They think you'r one of them!\n If you meet someone out there teach them what you know. It’s important that we preserve information.\n 7. If someone asks you for your hard earned equipment though, just give them a lit stick of dynamite\n 8. Watch out for wolves. Keep an eye out for cougars. Don't fuck with bears.\n 9. A water purifier is the best thing you can find. Clean water is precious.\n 10. Children zombies LOVE candy\n\""
```
Comment (I-am-Erk) [inline data/json/snippets/snippets.json]: I stand by my earlier statement: this is too much for any single survivor to put in a note at the start of the game, and would be better suited to a couple notes, or held off until we can adjust spawning to make it arrive later, with better reading options.

**Comment by kevingranade** [comment]
This pull request has been mentioned on **Cataclysm: Dark Days Ahead**. There might be relevant details there:

https://discourse.cataclysmdda.org/t/expanding-lore-and-detail-update-8-4-13/334/79

File: data/json/snippets/snippets.json (hunk 5 above)
Comment (rtizzy) [reply data/json/snippets/snippets.json]: @I-am-Erk 

If you would like to implement said feature or have someone else implement said feature, and then add this lore in, please don't hesitate to do so. 

Otherwise feel free to remove it from this pull

File: data/json/snippets/snippets.json (hunk 3 above)
Comment (rtizzy) [reply data/json/snippets/snippets.json]: Corrected

File: data/json/snippets/snippets.json (hunk 4 above)
Comment (rtizzy) [reply data/json/snippets/snippets.json]: Corrected

File: data/json/snippets/snippets.json (hunk 3 above)
Comment (rtizzy) [reply data/json/snippets/snippets.json]: Corrected to news article

File: data/json/snippets/snippets.json (hunk 2 above)
Comment (rtizzy) [reply data/json/snippets/snippets.json]: Updated to flier. Corrected

File: data/json/snippets/snippets.json (hunk 1 above)
Comment (rtizzy) [reply data/json/snippets/snippets.json]: Corrected

File: data/json/snippets/snippets.json (hunk 5 above)
Comment (rtizzy) [reply data/json/snippets/snippets.json]: Removed.

**Comment by rtizzy** [comment]
Pushed change to fix formatting issues that were causing the build to fail. Should pass now.

File: src/computer.cpp [hunk 6]
```diff
@@ -961,11 +961,11 @@ GREETINGS CITIZEN. A BIOLOGICAL ATTACK HAS TAKEN PLACE AND A STATE OF \n\
 EMERGENCY HAS BEEN DECLARED. EMERGENCY PERSONNEL WILL BE AIDING YOU \n\
 SHORTLY. TO ENSURE YOUR SAFETY PLEASE FOLLOW THE STEPS BELOW. \n\
 \n\
-1. DO NOT PANIC. \n\
-2. REMAIN INSIDE THE BUILDING. \n\
-3. SEEK SHELTER IN THE BASEMENT. \n\
-4. USE PROVIDED GAS MASKS. \n\
-5. AWAIT FURTHER INSTRUCTIONS. \n\
+1. PLEASE REMAIN CALM \n\
```
Comment (Night-Pryanik) [inline src/computer.cpp]:
```suggestion
1. PLEASE REMAIN CALM. \n\
```

**Comment by kevingranade** [comment]
Seems to be stalled, feel free to re-open if you want to address the outstanding issues.
